src/Matchers/DictionaryMatch.php

```diff
@@ -106,6 +106,8 @@ public function getFeedback($isSoleMatch)
     public function getFeedbackWarning($isSoleMatch)
     {
         switch ($this->dictionaryName) {
+            case 'user_inputs':
+                return 'This is easy to guess based on the other inputs';
```

Comment:
How about `'This is similar to, or incorporates parts of, other input'` ? I think that matches the other feedback language a better.